.editorconfig

```diff
@@ -1,3 +1,2 @@
 [*.{kt,kts}]
-max_line_length=210
```

Comment:
not needed?

Reply:
fixed as part of this PR (to make all the tools happy), see the few kt changes